seacrowd/sea_datasets/thai_nner/thai_nner.py

```diff
@@ -0,0 +1,594 @@
+    editor = "Muresan, Smaranda  and
+      Nakov, Preslav  and
+      Villavicencio, Aline",
+    booktitle = "Findings of the Association for Computational Linguistics: ACL 2022",
+    month = may,
+    year = "2022",
+    address = "Dublin, Ireland",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2022.findings-acl.116",
+    doi = "10.18653/v1/2022.findings-acl.116",
+    pages = "1473--1486",
+    abstract = "This paper presents the first Thai Nested Named Entity Recognition (N-NER) dataset.\
+Thai N-NER consists of 264,798 mentions, 104 classes,\
+and a maximum depth of 8 layers obtained from 4,894 documents in the domains of news articles and restaurant reviews.\
+Our work, to the best of our knowledge, presents the largest non-English N-NER dataset and\
+the first non-English one with fine-grained classes.\
+To understand the new challenges our proposed dataset brings to the field,\
+we conduct an experimental study on (i) cutting edge N-NER models with the state-of-the-art accuracy in English and\
+(ii) baseline methods based on well-known language model architectures.\
+From the experimental results, we obtained two key findings.\
+First, all models produced poor F1 scores in the tail region of the class distribution.\
+There is little or no performance improvement provided by these models with respect to the baseline methods with\
+our Thai dataset.\
+These findings suggest that further investigation is required to make\
+a multilingual N-NER solution that works well across different languages.",
+}
+"""
+
+
+_DATASETNAME = "thai_nner"
+
+
+_DESCRIPTION = """\
+Thai N-NER consists of 264,798 mentions, 104 classes,\
+and a maximum depth of 8 layers obtained from 4,894 documents in the domains of news articles and restaurant reviews.\
+To create the dataset,\
+the authors gather 4,894 documents from two different domains: news articles and restaurant reviews.
+In particular, \
+we obtain 4,396 news articles from Prachathai, a news website, and 498 restaurant reviews from Wongnai, a crowd-sourced restaurant review platform."""
+
+
+_HOMEPAGE = "https://github.com/vistec-AI/Thai-NNER"
+
+
+_LANGUAGES = ["tha"]
+
+
+_LICENSE = Licenses.CC_BY_SA_3_0.value
+
+
+_LOCAL = False
+
+
+_URLS = {"train": "https://drive.google.com/uc?export=download&id=1FNJ0Ylcq27SowSiMEIRzR--45Sz7dsGw", "test": "https://drive.google.com/uc?export=download&id=1FSM68Jh2Am4WByxrzzz4Cpou0J9Uok7Y"}
+
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class ThaiNnerDataset(datasets.GeneratorBasedBuilder):
+    """Thai Nested Named Entity Recognition (N-NER) dataset"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    label_classes = [
+        "O",
+        "S-org:religious",
+        "E-tv_show",
+        "I-soi",
+        "E-last",
+        "E-fund",
+        "B-country",
```

Comment:
While I'm checking, I also found this error `ValueError: Invalid string class label S-museum`, so far I noticed there are

```
        "S-hotel",
        "B-postcode",
        "E-postcode",
        "B-season",
        "E-season",
       ....
```

and probably more labels that aren't present in `label_classes`.

Could you please make sure the labels are complete and run `python3 -m tests.test_seacrowd seacrowd/sea_datasets/thai_nner/thai_nner.py` again?

Thank you!